lib/bootsnap/load_path_cache/store.rb

```diff
@@ -91,6 +91,7 @@ def dump_data
         FileUtils.mv(tmp, @store_path)
       rescue Errno::EEXIST
         retry
+      rescue Errno::EROFS
```

Comment:
I'd prefer a more general solution. I think you can rescue `SystemCallError` instead.

If there is a file system permission issue or whatever, etc.

Reply:
Confirmed (https://www.honeybadger.io/blog/understanding-rubys-strange-errno-exceptions/) thanks and done.